README.md

```diff
@@ -48,7 +48,10 @@ No. We continue to consume Fluent Bit from its main repository. We are not forki
 
 ### Public Images
 
-Each release updates the `latest` tag and adds a tag for the version of the image.
+Each release updates the `latest` tag and adds a tag for the version of the image. A `stable` tag is also available which can be trusted that it is the latest version in which there are no high impact bugs. A release may be marked as stable if the following rules are all met:
+* It has been out for a period of time
```

Comment:
I feel this is too vague. May be we should just say that our criteria is at least 2 weeks?